api/router.go

```diff
@@ -120,7 +120,11 @@ func NewRouter(s *Server) *mux.Router {
 
 	r.Methods("Get").Path("/v1/events").Handler(f(schemas, s.EventList))
 
-	r.Methods("Get").Path("/v1/supportbundle").Handler(f(schemas, s.GenerateSupportBundle))
+	r.Methods("POST").Path("/v1/supportbundles").Handler(f(schemas, s.InitiateSupportBundle))
+	r.Methods("GET").Path("/v1/supportbundles/{name}").Handler(f(schemas,
```

Comment:
Seems we can move those to the `/v1/nodes`, then it would be more consistent with existing API schema (since NodeID is required).

Add actions `supportBundleRequest`, `supportBundleQuery` and `supportBundleDownload` to the node API. UI can still use `POST /v1/supportbundles` to generate a support bundle without specifying a node.